lua/weapons/arccw_base/sh_util.lua

```diff
@@ -2,12 +2,16 @@ function SWEP:TableRandom(table)
     return table[math.random(#table)]
 end
 
-function SWEP:MyEmitSound(fsound, level, pitch, vol, chan)
+function SWEP:MyEmitSound(fsound, level, pitch, vol, chan, useWorld)
     fsound = self:GetBuff_Hook("Hook_TranslateSound", fsound) or fsound
 
     if istable(fsound) then fsound = self:TableRandom(fsound) end
 
     if fsound != "" then
-        self:EmitSound(fsound, level, pitch, vol, chan or CHAN_AUTO)
+		if useWorld then
+			sound.Play(fsound, self:GetPos() + VectorRand(), level, pitch, vol)
```

Comment:
Why the VectorRand? Also, I'm not sure what self:GetPos() would result in, but you should probably do owner:EyePos()

Reply:
sure